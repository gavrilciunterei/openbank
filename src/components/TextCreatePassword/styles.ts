import styled from 'styled-components';
import colors from '../../styles/colors';

export const BottomLine = styled.div`
  width: 50px;
  height: 5px;
  background-color: ${colors.blue_line};
  margin-top: -20px;
  margin-bottom: 20px;
`;
