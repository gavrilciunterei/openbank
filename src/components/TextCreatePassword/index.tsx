import React from 'react';
import Text from '../Text';
import { BottomLine } from './styles';
import ContentContainer from '../ContentContainer';
import { TextProps } from './types';
function TextCreatePassword({ text }: TextProps) {
  return (
    <ContentContainer>
      <Text fontWeight={500} size="30px">
        {text}
      </Text>
      <BottomLine />
    </ContentContainer>
  );
}

export default TextCreatePassword;
