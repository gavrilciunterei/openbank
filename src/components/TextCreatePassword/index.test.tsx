import React from 'react';
import renderer from 'react-test-renderer';
import TextCreatePassword from './index';

it('Test TextCreatePassword', () => {
  const tree = renderer
    .create(<TextCreatePassword text="Prueba texto cabecera" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
