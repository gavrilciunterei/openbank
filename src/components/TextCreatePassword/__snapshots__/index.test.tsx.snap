// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TextCreatePassword 1`] = `
<div
  className="sc-dkPtRN kBTENw"
>
  <p
    className="sc-bdvvtL jlpmb"
    fontWeight={500}
    size="30px"
  >
    Prueba texto cabecera
  </p>
  <div
    className="sc-gsDKAQ leMPoW"
  />
</div>
`;
